indigo_app/forms.py

```diff
@@ -656,6 +656,11 @@ def clean_spreadsheet_url(self):
         return re.sub('/[\w#=]*$', '/', url)
 
 
+class PlaceUsersForm(forms.Form):
+    choices = [(u.id, (u.get_full_name() or u.username)) for u in User.objects.filter(badges_earned__slug='editor')]
```

Comment:
This must be done when the form is initialised. Otherwise it is done once when this python file is first loaded, and never again.

Reply:
Or, I think the `choices` param below can be a lambda

Reply:
Thanks. Added.